src/main/java/net/william278/velocitab/packet/ScoreboardManager.java

```diff
@@ -74,14 +74,72 @@ public void resetCache(@NotNull Player player) {
         }
     }
 
+    public void vanishPlayer(Player player) {
+        if (!plugin.getSettings().doNametags()) {
```

Comment:
Is this related to the nametags feature? Can prob remove this check

Reply:
Yes maybe sorting